github/reactions.go

```diff
@@ -568,3 +568,54 @@ func (s *ReactionsService) CreateReleaseReaction(ctx context.Context, owner, rep
 
 	return m, resp, nil
 }
+
+// ListReleaseReactions lists the reactions for an release.
```

Comment:
```suggestion
// ListReleaseReactions lists the reactions for a release.
```

Reply:
@gmlewis [fixed](https://github.com/google/go-github/pull/3533/commits/7193318a031339961f7668c9d346fe55914aa5e6) (and another couple copy/paste typos)